CTP/shell/src/com/navercorp/cubridqa/shell/common/CommonUtils.java

```diff
@@ -120,12 +124,20 @@ public static String concatFile(String p1, String p2) {
 		return p.replace('/', File.separatorChar);
 	}
 	
-	public static String getFileNameForDispatchAll(){
-		return CommonUtils.concatFile(Constants.DIR_CONF, "dispatch_tc_ALL.txt");
+	public static String getFileNameForDispatchAll(Context context){
```

Comment:
Move below two methods to Dispatch.java

public static String getFileNameForDispatchAll(Context context)
public static String getFileNameForDispatchFin(String envId, Context context){